app/views/provider/admin/dashboards/_service_navigation.html.slim

```diff
@@ -37,21 +36,21 @@ nav.DashboardNavigation
       li.DashboardNavigation-list-item
         // Subscriptions (would be nice to have API specific links)
         => dashboard_collection_link 'Subscription',
-                                     service.service_contracts,
+                                     service.service_contracts_count,
                                      admin_buyers_service_contracts_path(search: {service_id: service.id}),
                                      icon_name: 'paperclip'
 
     // Active docs
     - if can?(:manage, :plans)
       li.DashboardNavigation-list-item
        = dashboard_collection_link 'ActiveDoc',
-                                    service.api_docs_services,
+                                    service.api_docs_services_count,
                                     admin_service_api_docs_path(service),
                                     icon_name: 'file-text'
     // Integration
     - if can?(:manage, :plans)
       li.DashboardNavigation-list-item
-        - if service.has_traffic? || (apiap? && service.proxy.proxy_configs.sandbox.any?)
+        - if service.cinstances_with_traffic_count > 0 || (apiap? && service.sandbox_proxy_configs_count > 0)
```

Comment:
Are you sure that this performs better than this?:
```ruby
        - if service.cinstances_with_traffic.exists? || (apiap? && service.sandbox_proxy_configs.exists?)
```
Because the `exists?` works with `LIMIT 1` which implies that instead of counting them all, it only checks until it find the first one if there is any 😄

Reply:
so i finally have an asnwer: https://issues.redhat.com/browse/THREESCALE-3592?focusedCommentId=14116380&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-14116380